lua/customroles/admin.lua

```diff
@@ -20,6 +20,7 @@ You can use your admin menu to access commands
 that will help in the battle against the {traitors}.
 
 Press {menukey} to receive your equipment!]]
+ROLE.shortdesc = "Has a menu of admin commands that they can spend thier regenerating power on to use against other players to help or to hurt."
```

Comment:
```suggestion
ROLE.shortdesc = "Has a menu of admin commands that they can use against other players to help or to hurt."
```
I feel like this is a bit too wordy and I would probably simplify by omitting the admin power bit